### DISCUSSION TITLE
How to install extensions with bookmarklet

### DISCUSSION THREAD

**Discussion by phillipbruh** [3kh0/ext-remover#144] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Option 1(better)
1. make sure you have more than google account signed in
2. goto https://chrome.google.com/webstore?authuser=1
3. now you can browse freely

Option 2(bookmarklet)
This will not work if the extension is blocked this is only if the webstore is blocked

1.on a diffrent pc get the exstenstion id you can get it by looking in the url and it should be after the name so exname/132294420
2. then goto this link and run the replit
https://replit.com/@CompactCow/i-love-replit-it-is-my-favorite-website-ever-and-its-so-goo?v=1
3. put in your id and click enter
4. copy the bookmarklet it gives you and run it
5. it should install the extension

**Comment by ghost** [comment]
k

**Comment by The-Man-Is-Me** [comment]
it gives me this, any help??
https://storage.googleapis.com/replit/images/1667000640531_e6bfaa7030a340bc4660790e2fc13fa5.png

**Reply by 3kh0** [reply]
@The-Man-Is-Me do not fork it, just run it as it is. You also might have included a space or a extra thing in your input

**Reply by The-Man-Is-Me** [reply]
yes, i did put it wrong, however when done correctly it gives me this
![Screenshot 2022-10-28 7 20 27 PM](https://user-images.githubusercontent.com/78439704/198753104-bedb548f-a717-47cd-b379-160166ad73ab.png)

**Reply by 3kh0** [reply]
Quoting from the post above:
> not this will not work if the extension is blocked this is only if the webstore is blocked

Please read

**Comment by brobcoolguy** [comment]
i created a account just so i can say thanks, i desperately needed this. so thanks

**Comment by lazystudentafk** [comment]
the replit link has a 404 error

**Reply by 2pro12342** [reply]
> the replit link has a 404 error

probably because this was 4 years ago and they didn't want the website up that long

[a comment by an automated account is not shown]
